### ISSUE TITLE
travis show failed

### ISSUE THREAD

**Issue by kpp** [travis-ci/travis.rb#282]
```
$ travis show
Build #6:  Try ubuntu toolchain repo
State:         started
Type:          push
Branch:        feature/65/travis
Compare URL:   https://github.com/0x7CFE/llst/compare/705340acf4de...249968479440
Duration:      none
Started:       2015-03-20 00:13:59
Finished:      not yet

undefined method `to_sym' for nil:NilClass
for a full error report, run travis report --org
```

```
$ travis report --org
System
Ruby:                     Ruby 1.9.3-p484
Operating System:         LinuxMint 17
RubyGems:                 RubyGems 1.8.23

CLI
Version:                  1.7.5
Plugins:                  none
Auto-Completion:          yes
Last Version Check:       2015-03-20 00:06:10 +0300

Session
API Endpoint:             https://api.travis-ci.org/
Logged In:                as "kpp"
Verify SSL:               yes
Enterprise:               no

Endpoints
org:                      https://api.travis-ci.org/ (access token, current)

Last Exception
An error occurred running `travis show --org`:
    NoMethodError: undefined method `to_sym' for nil:NilClass
        from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli/command.rb:312:in `map'
        from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli/command.rb:312:in `color'
        from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli/show.rb:32:in `block in run'
        from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli/show.rb:30:in `each'
        from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli/show.rb:30:in `run'
        from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli/command.rb:198:in `execute'
        from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli.rb:64:in `run'
        from /var/lib/gems/1.9.1/gems/travis-1.7.5/bin/travis:18:in `<top (required)>'
        from /usr/local/bin/travis:23:in `load'
        from /usr/local/bin/travis:23:in `<main>'
```

**Comment by kpp** [comment]
```
$ travis show 13
/usr/lib/ruby/1.9.1/erb.rb:838:in `eval': (erb):2: syntax error, unexpected ')', expecting tSTRING_CONTENT or tSTRING_DBEG or tSTRING_DVAR or tSTRING_END (SyntaxError)
...t.concat(( color("received", :) ).to_s); _erbout.concat "\n"
...                               ^
	from /usr/lib/ruby/1.9.1/erb.rb:838:in `result'
	from /var/lib/gems/1.9.1/gems/highline-1.7.1/lib/highline.rb:721:in `format_statement'
	from /var/lib/gems/1.9.1/gems/highline-1.7.1/lib/highline.rb:621:in `say'
	from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli/command.rb:248:in `say'
	from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli/show.rb:26:in `run'
	from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli/command.rb:198:in `execute'
	from /var/lib/gems/1.9.1/gems/travis-1.7.5/lib/travis/cli.rb:64:in `run'
	from /var/lib/gems/1.9.1/gems/travis-1.7.5/bin/travis:18:in `<top (required)>'
	from /usr/local/bin/travis:23:in `load'
	from /usr/local/bin/travis:23:in `<main>'
```
Wait for few seconds...
```
$ travis show 
Build #13:  Traces ld version
State:         started
Type:          push
Branch:        feature/65/travis
Compare URL:   https://github.com/0x7CFE/llst/compare/b7609e4acd46...c1696ed92121
Duration:      none
Started:       2015-03-20 12:34:17
Finished:      not yet

#13.1 started:   1 sec          compiler: gcc, env: USE_LLVM=Off, os: linux
#13.2 started:   1 sec          compiler: gcc, env: USE_LLVM=On, os: linux
```
And try again:
```
$ travis show 13
Build #13:  Traces ld version
State:         started
Type:          push
Branch:        feature/65/travis
Compare URL:   https://github.com/0x7CFE/llst/compare/b7609e4acd46...c1696ed92121
Duration:      none
Started:       2015-03-20 12:34:17
Finished:      not yet

#13.1 started:   7 sec          compiler: gcc, env: USE_LLVM=Off, os: linux
#13.2 started:   7 sec          compiler: gcc, env: USE_LLVM=On, os: linux
```

**Comment by kpp** [comment]
Seems to be fixed in `1.7.6`
